### PR TITLE
User login check wrapped with eventually

### PR DESCRIPTION
**Pull request by SergeyKemaev**
Title is self-descriptive
Fix for https://github.com/qubell/vermilion/issues/6885

Risk: Low

Impact: Test code fixed to not assume that instance is completely ready even if it's Active.

Observable effect: monitor check will not fail just because instance is not completely initialized (web server hasn't yet started web app)

### CODE REVIEW

**Comment by SergeyKemaev** [comment]
@dornatsky at last this PR has been built successfully. Let's merge?

File: test/test.py [hunk 1]
```diff
@@ -83,5 +93,9 @@ def test_check_user_login(self, instance):
       timeout = 10
       socket.setdefaulttimeout(timeout)
       url = instance.returnValues['Cloudera.manager']
-      response = requests.get(url, auth=requests.auth.HTTPBasicAuth('admin', 'admin'))
-      self.assertEqual(200, response.status_code)
+      
+      @eventually(AssertionError)
+      def eventually_assert():
+        response = requests.get(url, auth=requests.auth.HTTPBasicAuth('admin', 'admin'))
```
Comment (dornatsky) [inline test/test.py]: what are these creds?

**Comment by dornatsky** [comment]
Shouldn't tests kinda expect it to be completely initialized? Are we talking about bigger timeouts here?

File: test/test.py (hunk 1 above)
Comment (SergeyKemaev) [inline test/test.py]: These creds are defaults from manifest. Ideally they should be part of tests configuration of course.

**Comment by SergeyKemaev** [comment]
Tests expect that instance in status `Active` is completely initialized. However this status does not mean that webserver has been start up completely, so yes, we need bigger timeout for test that checks possibility to login.

**Pull request merged**